resources/build/build-utils-ci.test.sh

```diff
@@ -8,50 +8,50 @@ set -eu
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
-. "${THIS_SCRIPT%/*}/build-utils.sh"
+. "${THIS_SCRIPT%/*}/../../resources/build/build-utils.sh"
```

Comment:
Why the path change here and in similar scripts in this folder?

Reply:
Noted in comment for 350808bd5d6a644310865a453b925097766df1f8, and documented here:

https://github.com/keymanapp/keyman/blob/3302abaf73b732809e3612a57cfbfe843b81f136/resources/build/build-utils.md?plain=1#L94-L96

More detail: this means we can eliminate further `readlink` calls to determine the root because we know we are starting at `/resources`; it's only an issue for scripts under `resources/` folder because all others will already need to reference `/resources` to be able to get this script.

Not excited about this but it's a good optimization otherwise!

Reply:
... how is this not still relative to the source script's path?  The include path is still 100% based on it - the whole `../..` bit is _relative_, after all.  Only after that relative op do we when reconstruct the path.

Note:  it's only a complaint about cases matching this specific use case - the inclusion of `build-utils.sh`.

Reply:
It's still relative but it includes the repo root, which is necessary for the fast path parsing we are using (removing path components without canonicalization).

I am thinking of rolling this optimization back because it's non-obvious and liable to lead to pain in the future. May make the build scripts a bit slower but there are plenty of other wins in this PR anyway.